build.gradle

```diff
@@ -411,7 +411,8 @@ dependencies {
 
     compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.39'
     compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.39'
-    compile group: 'org.hibernate', name: 'hibernate-core', version: '5.4.19.Final'
+    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.13'
+    compile group: 'org.hibernate', name: 'hibernate-core', version: '5.4.24.Final'
```

Comment:
I think, we can remove all the hibernate related dependency from ORM as there is no DB operation.